tasks.py

```diff
@@ -132,7 +129,7 @@ def lint(context):
         Skips init.py files for import sorts
     """
     subprocess.run(["flake8", "pyreal", "tests"], check=True)
-    subprocess.run(["isort", "-c", "pyreal", "tests", "--skip", "__init__.py"], check=True)
+    subprocess.run(["isort", "-c", "pyreal", "tests"], check=True)
```

Comment:
Replace `-c` with `src_paths = ["pyreal", "tests"]` inside the [config in pyproject.toml](https://pycqa.github.io/isort/docs/configuration/config_files.html)